internals-js/src/error.ts

```diff
@@ -555,6 +555,112 @@ const INTERFACE_KEY_MISSING_IMPLEMENTATION_TYPE = makeCodeDefinition(
   { addedIn: '2.3.0' },
 )
 
+const SOURCE_API_NAME_INVALID = makeCodeDefinition(
```

Comment:
All these should be `2.7.0`

Reply:
https://github.com/apollographql/federation/pull/2910/commits/0736da204b9b236c36233d9d330395b6c9a17c79